ncm-dirperm/src/main/pan/components/dirperm/schema.pan

```diff
@@ -30,11 +30,13 @@ function dirperm_permissions_valid = {
 };
 
 type structure_dirperm_entry = {
-    'path'    : string with match(SELF, '^/')
-    'perm'    : string with match(SELF,'[0-7]{3,4}')
-    'owner'   : string with match(SELF, '\w+(:\w+)?')
-    'type'    : string with match(SELF, 'd|f')
-    'initdir' ? string[]
+    'path'       : string with match(SELF, '^/')
+    'perm'       : string with match(SELF,'[0-7]{3,4}')
+    'owner'      : string with match(SELF, '\w+(:\w+)?')
+    'type'       : string with match(SELF, 'd|f')
+    'initdir'    ? string[]
+    'checkmount' : boolean = false
+    'notroot'    : boolean = false
```

Comment:
We discussed this here the other day. 

I agree notroot=false and root=true are the same thing, but I disagree that as a new attribute it won't affect anyone. a root=true default will break the creation of directories in mounts, where people are already doing this.

I've changed it to "within_mount", I think this was suggested by @ned21, to avoid the double negation. 

"notroot" or "within_mount" only lets us check for one possible configuration (instead of two - non-root and root), so avoids the problem. Hope this explains it better :)